apps/communications/contacts/elements/form.html

```diff
@@ -127,11 +127,11 @@ <h1 id='contact-form-title' data-l10n-id="addContact"></h1>
             </div>
             <div class="item-body-exp with-overflow">
               <p class="fillflow-row input-container">
-                <input placeholder="Name" data-field="givenName" data-l10n-id="givenName" name="givenName" class="textfield" type="text" x-inputmode="latin-prose" value="" id="givenName" dir="auto">
+                <input placeholder="Name" data-field="givenName" data-l10n-id="givenName" name="givenName" class="textfield" type="text" x-inputmode="latin-prose" value="" id="givenName" dir="auto" autocorrect="off" autocapitalize="off" spellcheck="false" x-inputmode="verbatim">
```

Comment:
I think the autocapitalize off is not a good idea. Perhaps in some cases we want to have it in lower case, but 90% of the cases a name and surname will be capitalized